client/src/pages/Home/Home.jsx

```diff
@@ -30,6 +32,23 @@ const Home = ({ signIn, signUp, setisAuthenticated, isAuthenticated }) => {
     console.log("Here 1");
   }, []);
 
+  const productFilters = (filterType, filter) => {
+    const currentFilteredProducts = [];
+    if (filterType === "price") {
+      products.map((product) => {
+        if (
+          parseInt(product.price) > parseInt(filter[0]) &&
```

Comment:
Line 40 and 41 are causing the filter to not return products that lie on the parameters of the filter. For example if we filter by price from $2 to $500 products that are $2 and $500 will not show up in the filtered results. To fix this just change the relational operators on lines 40 and 41 to ">=" and "<=" instead of ">" and "<"